src/components/listing/ListingPage.jsx

```diff
@@ -1,170 +1,74 @@
 import React from "react";
+import { useData } from "../../context/DataContext/DataContext";
 import TrendingCard from "../product-card/TrendingCard/TrendingCard";
 import ProductCardVertical from "../product-card/VerticalCard/ProductCardVertical";
 
 import "./ListingPage.css";
 
-const ListingPage = ({ item }) => {
+const ListingPage = () => {
+  const { data } = useData();
+
+  let trendingArr = [];
+  let productsArr = [];
+
+  data.products.forEach((item) => {
+    item.trendingText === "Trend"
+      ? trendingArr.push(item)
+      : productsArr.push(item);
+  });
+
   return (
-    <main class="products-content">
-      <p class="text-regular-weight text-left">Products</p>
+    <main className="products-content">
+      <p className="text-regular-weight text-left">Products</p>
       <p>(Showing all products)</p>
-      <TrendingCard
-        item={{
-          productName: "Blue T-Shirt",
-          productShortDescription: "Lorem Ipsum has been the dummy text",
-          productLongDescription:
-            "Lorem Ipsum has been the dummy text Lorem Ipsum has been the dummy text",
-          trendingText: "Trend",
-          categoryName: "men",
-          soldBy: "Amazon",
-          rating: 4.4,
-          salePrice: 899,
-          originalPrice: 999,
-          discountPercent: 10,
-          closeActive: false,
-          wishlisted: false,
-          imagePath: "/assets/product.jpg",
-        }}
-      />
+      {trendingArr.map((item) => {
+        return (
+          <TrendingCard
+            key={item._id}
+            item={{
+              id: item._id,
+              productName: item.productName,
+              productShortDescription: item.productShortDescription,
+              productLongDescription: item.productLongDescription,
+              trendingText: item.trendingText,
+              categoryName: item.categoryName,
+              soldBy: item.soldBy,
+              rating: item.rating,
+              salePrice: item.salePrice,
+              originalPrice: item.originalPrice,
+              discountPercent: item.discountPercent,
+              closeActive: item.closeActive,
+              wishlisted: item.wishlisted,
+              imagePath: item.imagePath,
```

Comment:
Check whether it is good or destructuring item in component is good...

Reply:
Will look into this and update in fixes feature